### DISCUSSION TITLE
Hacktoberfest is over

### DISCUSSION THREAD

**Discussion by atompie** [Tracardi/tracardi#763] [General]
The hacktoberfest is over we would like to thank all the participants and contributors to our http://github.com/tracardi/tracardi project.

We have merged and accepted 37 PRs. If any of you would like to continue to contribute to the project you are all welcome. 
 
@knittl @Mefpef @madhavmehndiratta @hyprex-deva @CypherpunkSamurai @sameersecond @ojninja16 @yk-jp @sameersecond @tanayseven

**Comment by tanayseven** [comment]
I would like to contribute to a couple more issues, just to see if I enjoy this project or not

**Comment by CypherpunkSamurai** [comment]
> If any of you would like to continue to contribute to the project you are all welcome.

Sure, I usually contribute to FOSS issues that I find comfortable with and unassigned. I'll definitely keep an eye for them so I can help the project 😄
